auth/auth-router.js

```diff
@@ -23,6 +23,7 @@ router.post('/login', (req, res) => {
   Users.getUserBy({username})
   .then(user => {
     if(user && bcrypt.compareSync(password, user.password)){
+      delete user.rememberMe;
```

Comment:
I'm confused by this... won't this prevent the token from ever seeing rememberMe? Shouldn't this be deleted after the call to generateToken?